### ISSUE TITLE
Docker image for patching fonts

### ISSUE THREAD

**Issue by carlosedp** [ryanoasis/nerd-fonts#422]
I've recently patched some fonts and created a Docker image with all requirements installed so there is no need to install all tools into your own box.

The image is pushed into <https://hub.docker.com/r/carlosedp/nerdfonts-patch> and can be run with the following command in the same dir as the font:

`docker run -it --rm -v $(pwd):/fonts carlosedp/nerdfonts-patch [font file]`

The patched font will be placed in the same directory.

If you want to build the Dockerfile yourself, the file is:

```dockerfile
# Build with docker build -t carlosedp/nerdfonts-patch -f Nerdfonts-patch.dockerfile .
# Run with docker run -it --rm -v $(pwd):/fonts carlosedp/nerdfonts-patch [font file]

FROM ubuntu:14.04

WORKDIR /root

RUN apt-get update && \
    sudo apt-get install -y software-properties-common && \
    sudo add-apt-repository -y ppa:fontforge/fontforge && \
    sudo apt-get update && \
    sudo apt-get install -y fontforge python python-pip python-fontforge git

RUN git clone https://github.com/ryanoasis/nerd-fonts --depth=1 && \
    cd nerd-fonts && \
    rm -rf .git patched-fonts src/unpatched-fonts

RUN pip install configparser

RUN locale-gen en_US.UTF-8 && \
    dpkg-reconfigure locales

ENV LC_ALL=en_US.UTF-8
ENV LANG=en_US.UTF-8
ENV LANGUAGE=en_US.UTF-8

WORKDIR /fonts

ENTRYPOINT ["/root/nerd-fonts/font-patcher", "--careful", "--complete", "--progressbar", "--out", "/fonts/"]

```

To build the image, use the command `docker build -t [yourrepo]/nerdfonts-patch -f Nerdfonts-patch.dockerfile .`

Is there interest to add this to the Nerd fonts repo?

**Comment by ryanoasis** [comment]
> Is there interest to add this to the Nerd fonts repo?

Yes, for sure. I feel like the more options (especially if they make things easier) for people to have the better and more interest there'd be.

**Comment by ryanoasis** [comment]
Considering this solved with #449 
Feel free to submit any further PRs to improve it though. Thanks for getting the ball rolling!
